doc/nrf/ug_partition_manager.rst

```diff
@@ -69,18 +69,97 @@ placement: dict{}
       after: list
          Place the partition after the first existing partition in the list.
 
-   The valid values in the lists are `app`, `end`, or the name of any partition.
-   It is not possible to place the partition after `end`.
-
-sub_partitions: list
-   Define sub partitions which span across one or more other partitions.
-   The sub partitions split the total size of the spanned partitions, and therefore have equal size.
-   This property cannot be used together with the `placement` property.
-   When this property is set for a partition, it is required that the `span` property is also set.
+   The valid values in the lists are `app`, `start`, `end`, or the name of any partition.
+   It is not possible to place the partition after `end` or before `start`.
 
 span: list
-   When creating sub_partitions, this option lists what partitions the sub_partitions should span across.
+   This property lists what partitions this partition should span across.
    This property cannot be used together with the `placement` property.
+   Non-existing partitions are ignored.
+   The Partition Manager will fail if a span is empty, or the partitions are not consecutive after processing.
```

Comment:
Add a comment about what the listings below contains.

"The samples below show examples of ..."

Reply:
Added some text